### DISCUSSION TITLE
Welcome to BlackBox Discussions!

### DISCUSSION THREAD

**Discussion by FBlackBox** [FBlackBox/BlackBox#123] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by CarryFan** [comment]
诚招插件化相关有志之士加入我们，共谋大业~
vx手机同号：17371032688

**Comment by cyc0203** [comment]
请问，能否支持安卓13

**Reply by w1723423229** [reply]
大佬 安卓13麻烦适配一下

**Reply by kimnanchen** [reply]
大佬 安卓13麻烦适配一下

**Comment by Hubert66632** [comment]
大佬，请修复blackbox在vivo手机无法创建快捷方式的bug，拜托了 @FBlackBox

**Reply by Hubert66632** [reply]
@FBlackBox 拜托你看到这个留言

**Reply by Yea8m** [reply]
Android13 应用闪退，请修复一下，Thank you！Gratitude!!

**Reply by ghost** [reply]
Hi Brother android 13 Facebook not open
